transmogrifier/sources/json/aardvark.py

```diff
@@ -385,16 +385,36 @@ def get_publication_information(source_record: dict) -> list[str]:
         return publication_information
 
     @staticmethod
-    def get_rights(source_record: dict) -> list[timdex.Rights]:
+    def get_rights(source: str, source_record: dict) -> list[timdex.Rights]:
         """Get values from source record for TIMDEX rights field."""
         rights = []
+        kind_access_to_files = "Access to files"
```

Comment:
The variable name is longer than the actual string so it feels unnecessary to use a variable

Reply:
On second thought, that's fine. Disregard my comment